org/mozilla/jss/pkcs11/PK11Signature.c

```diff
@@ -355,6 +454,81 @@ getAlgorithm(JNIEnv *env, jobject sig)
     return retval;
 }
 
+static SECOidTag
+getDigestAlgorithm(JNIEnv *env, jobject sig)
+{
+    jclass sigClass;
+    jfieldID algField;
+    jobject alg;
+    SECOidTag retval=SEC_OID_UNKNOWN;
+
+    PR_ASSERT(env!=NULL && sig!=NULL);
+
+    sigClass = (*env)->GetObjectClass(env, sig);
+    PR_ASSERT(sigClass != NULL);
+
+    algField = (*env)->GetFieldID(  env,
+                                    sigClass,
+                                    SIG_DIGEST_ALGORITHM_FIELD,
+                                    SIG_ALGORITHM_SIG);
+    if(algField == NULL) {
+        ASSERT_OUTOFMEM(env);
+        goto finish;
+    }
+
+    alg = (*env)->GetObjectField(env, sig, algField);
+    if(alg == NULL) {
+        ASSERT_OUTOFMEM(env);
+        goto finish;
+    }
+
+    retval = JSS_getOidTagFromAlg(env, alg);
+
+finish:
+    return retval;
+}
+
+static
+
```

Comment:
Spare newline.

Reply:
Done. I think. If I missed any of these, it was an over sight... :)